src/CommonITILObject.php

```diff
@@ -8468,8 +8468,12 @@ public static function getDataToDisplayOnKanban($ID, $criteria = [])
         foreach ($iterator as $data) {
             // Create a fake item to get just the actors without loading all other information about items.
             $temp_item = new static();
-            $temp_item->fields['id'] = $data['id'];
+            $temp_item->getEmpty();
+            $temp_item->fields = array_merge($temp_item->fields, $data);
             $temp_item->loadActors();
+            if (!$temp_item->canViewItem()) {
```

Comment:
`Ticket::canViewItem()` may check `users_id_recipient` field value.

Instead of using `CommonDBTM::getEmpty()` and then merge query result into `$fields`, using a `SELECT *` would probably be more appropriate (and easier to maintain). As query result already contains `content`, which is likely to consume a lot more DB bandwith than all other fields combined, the DB bandwith overhead would probably not be a problem. Therefore, it would probably require data to be filtered before beiing sent to the browser.